pkg/stacks/unpack_test.go

```diff
@@ -246,15 +246,23 @@ metadata:
     stacks.crossplane.io/icon-data-uri: data:image/svg+xml;base64,bW9jay1pY29uLWRhdGEtc3Zn
     stacks.crossplane.io/stack-title: Sample Crossplane Stack
     stacks.crossplane.io/ui-schema: |-
-      version: 0.3
+      version: 0.4
```

Comment:
Do we normally change the design files and the go files in the same pull request? Not opposed to it, but I'm asking because I would usually expect them to happen in separate pull requests, and I want to see if I'm just not aware of a pattern we have.

Reply:
That is a good question and I am not sure I have an answer to it, it just felt appropriate to update this file. @jbw976 any thoughts here?